locales/locales.json

```diff
@@ -851,6 +851,26 @@
       "currentlyNoEvents": "Güncel bir etkinlik yok."
     }
   },
+  "news": {
+    "de": {
+      "local": "LOKALE NEWS",
+      "readMore": "Weiterlesen >",
+      "currentlyNoNews": "Es gibt aktuell keine Veranstaltungen.",
+      "eNewsNo": "E-news No"
+    },
+    "ar": {
+      "local": "الأخبار المحلية",
+      "currentlyNoNews": "ليست هناك أخبار متوفرة حاليا",
+      "readMore": "< المزيد",
+      "eNewsNo": "E-news No"
+    },
+    "en": {
+      "local": "LOCAL NEWS",
+      "readMore": "Read more >",
```

Comment:
Imo the arrow should not be part of the locales, what do you think?

Reply:
Yeah you are right